bin/testObservability/helper/helper.js

```diff
@@ -522,6 +535,7 @@ exports.uploadEventData = async (eventData, run=0) => {
       
       exports.requestQueueHandler.start();
       const { shouldProceed, proceedWithData, proceedWithUrl } = exports.requestQueueHandler.add(eventData);
+      exports.debugOnConsole(`[Request Queue] ${eventData.event_type} with uuid ${eventData.test_run ? eventData.test_run.uuid : (eventData.hook_run ? eventData.hook_run.uuid : null)} is added`)
```

Comment:
there is a return statement at the beginning of the method as well. Let's add a log at the start of `uploadEventData`